halyard-deploy/src/main/resources/kubernetes/manifests/deployment.yml

```diff
@@ -19,6 +19,16 @@ spec:
     matchLabels:
       app: spin
       cluster: spin-{{ name }}
+  {% if deploymentStrategy != null %}
+  strategy:
+    {% if deploymentStrategy.type == "rollingUpdate" and deploymentStrategy.strategySettings|length > 0 %}
+    {{ deploymentStrategy.type }}:
+      {% for key, value in deploymentStrategy.strategySettings.items() %}
+      {{ key }} : {{ value }}
+      {% endfor %}
+    {% endif %}
+    type: {% if deploymentStrategy.type == "recreate" %} Recreate {% else %} RollingUpdate {% endif %}
+  {% endif %}
```

Comment:
Minor thing, I changed the order here so that the default is `RollingUpdate` not `Recreate` since the current default for Kubernetes if not specified is `RollingUpdate`